src/app/code/community/Omise/Gateway/etc/system.xml

```diff
@@ -153,6 +153,27 @@
                                 </active>
                             </fields>
                         </omise_offsite_alipay>
+
+                        <omise_offsite_instalment type="group">
+                            <label>Instalment Payments</label>
```

Comment:
@jonrandy I vote to have only Installment just to make it consistent with the others.

Reply:
![image](https://user-images.githubusercontent.com/1510194/51970772-5031ec00-24aa-11e9-9e31-3e7df728333b.png)